README.md

```diff
@@ -1,8 +1,64 @@
+# Eclipse LSP4MP - Language Server for Eclipse MicroProfile
+
 [![Build Status](https://ci.eclipse.org/lsp4mp/buildStatus/icon?job=lsp4mp%2Fmaster)](https://ci.eclipse.org/lsp4mp/job/lsp4mp/job/master/)
+[![License](https://img.shields.io/badge/License-EPL%202.0-brightgreen.svg)](https://www.eclipse.org/legal/epl-2.0/)
+
+A language server implementation based on the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) for [Eclipse MicroProfile](https://microprofile.io/). This Language Server for Eclipse MicroProfile (LSP4MP) provides core language support capabilities (such as code complete, diagnostics, quick fixes) to enable developers to quickly and easily develop applications using MicroProfile APIs.

```

Comment:
I think it's very important to mention those capabilities depends on the user Java project ( ex: you have completion for properties for Microprofile Fault Tolerance only if the Java project have MP FT has dependency).

It will be more easy to explain why we need JDT. The main idea is that MicroProfile LS delegate to an external complement the job with Java (ex : collection of MP properties used in completion in microprofile-config.properties, Java diagnostics, etc)

external complement is managed with MicroProfile JDT used in vscode and Eclipse IDE context. But external complement can be implemented with another mean, like IJ Quarkus does (it uses only MicroProfile LS and not MicroProfile JDT).
IMHO we need a schema which explain the architecture (we could do that in an another PR).